RFC.md

```diff
@@ -1,12 +1,295 @@
 # Introducing a new multidimensional array indexing intrinsic
 
-# Intrinsic specification
+## The request for change: A new instruction, `multidim_array_index`.
```

Comment:
I am surprised you are going directly to a new instruction. Usually one introduces an intrinsic first, which in the future could be converted into an instruction if desired. See https://llvm.org/docs/ExtendingLLVM.html

The underscore does not reflect instruction naming rules. It's `getelementptr`, not `get_element_ptr`